site/public/css/electronic.css

```diff
@@ -1462,7 +1462,7 @@ tr.is_publish {
 }
 
 [id^=textbox-solution-] {
-    min-height: 120px;
+    min-height: 150px !important;
```

Comment:
does this work without the `!important`? I think using `!important` [might be bad for future maintainability](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity#how_is_specificity_calculated), but someone else might know about this better than I do